exercises/saddle-points/tests/saddle-points.rs

```diff
@@ -0,0 +1,68 @@
+extern crate saddle_points;
+
+use saddle_points::*;
+
+#[test]
+fn identify_single_saddle_point() {
+    let input: Vec<Vec<u64>> = vec![vec![9, 8, 7],
+                                    vec![5, 3, 2],
+                                    vec![6, 6, 7]];
+    assert_eq!(vec![(1,0)], find_saddle_points(input));
+}
+
+
+#[test]
+#[ignore]
+fn identify_empty_matrix() {
+    let input: Vec<Vec<u64>> = vec![vec![]];
+    let expected: Vec<(u64, u64)> = Vec::new();
+
+    assert_eq!(expected, find_saddle_points(input));
+}
+
+#[test]
+#[ignore]
+fn identify_missing_saddle_point() {
+    let input: Vec<Vec<u64>> = vec![vec![1, 2, 3],
+                                    vec![3, 2, 1],
+                                    vec![2, 3, 1]];
+    let expected: Vec<(u64, u64)> = Vec::new();
+    assert_eq!(expected, find_saddle_points(input));
+}
+
+#[test]
+#[ignore]
+fn identify_bottom_right_saddle_point() {
+    let input: Vec<Vec<u64>> = vec![vec![8, 7, 9],
+                                    vec![6, 7, 6],
+                                    vec![3, 2, 5]];
+    assert_eq!(vec![(2,2)], find_saddle_points(input));
+}
+
+#[test]
+#[ignore]
+fn non_quadratic_matrix_high() {
```

Comment:
Non-imperative quibble: change `*quadratic*` test names to use the term `square` instead. While "quadratic" isn't wrong, it has associations with the operation of squaring more than the shape of a matrix. Also, it's usually better to name things for clarity, and "square" is a much more common word.

If you feel strongly that the term should remain, it's fine; I don't feel that this is an essential change, just one I'd prefer.